openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -198,7 +206,7 @@ This is an example of a Wallet Instance Attestation:
 
 * The Credential Issuer MUST publish a mapping of every Credential Type it supports to a scope value.
 
-# OpenID for Verifiable Presentations
+# OpenID for Verifiable Presentations without using W3C Digital Credentials API
```

Comment:
Line 211 states that "MUST support protocol extensions for SD-JWT VC credential format profile". Does this mean this section (OpenID for Verifiable Presentations without using W3C Digital Credentials API) applies only to SD-JWT VC? If so, it should be made clear in the title.

Reply:
```suggestion
# OpenID for Verifiable Presentations profile for SD-JWT VC without using W3C Digital Credentials API
```